src/App.js

```diff
@@ -1,25 +1,20 @@
-import logo from './logo.svg';
-import './App.css';
+import useLocalStorageState from "use-local-storage-state";
+import { uid } from "uid";
+import Form from "./components/form/Form.js";
+
+export default function App() {
+  const [activities, setActivities] = useLocalStorageState("activities", {
+    defaultValue: [],
+  });
+
+  function handleAddActivity(newActivity) {
+    newActivity = { ...newActivity, id: uid() };
+    setActivities([newActivity, ...activities]);
```

Comment:
I recommend to not assign a new value to `newActivity`. You can simply do that all in `setActivities`:

```suggestion
    setActivities([{ ...newActivity, id: uid() }, ...activities]);
```